CHANGES.rst

```diff
@@ -5,6 +5,8 @@ Changelog
 Unreleased
 ----------
 
+* Updated permissions of the service account to be able to watch secrets.
+
```

Comment:
I don't think you need this changelog entry, since the change is part of https://github.com/crate/crate-operator/commit/a19d9447bec053b37c9deb35966b7f1e40b93dcc (which has not been released yet).

Reply:
Good point